setup.py

```diff
@@ -70,10 +66,11 @@ def get_version(verbose=1, filename='src/qtt/version.py'):
       package_dir={'': 'src'},
       packages=find_packages(where='./src'),
       install_requires=[
-          'matplotlib', 'pandas', 'attrs', 'dulwich', 'qtpy', 'nose', 'hickle', 'pyzmqrpc',
+          'matplotlib>=3.0', 'pandas', 'attrs', 'dulwich', 'qtpy', 'nose', 'hickle', 'pyzmqrpc',
           'numpy>=1.15', 'scikit-image', 'IPython>=0.1', 'qcodes>=0.4', 'Polygon3',
           'scipy', 'pyqtgraph', 'qupulse'
       ],
+      tests_require=['unittest'],
```

Comment:
```suggestion
      tests_require=['pytest'],
```

The module unittest comes with python preinstalled, pytest on the other hand not.